api/grpcserver/v2beta1/smeshing_test.go

```diff
@@ -0,0 +1,31 @@
+package v2beta1
+
+import (
+	"context"
+	"testing"
+
+	spacemeshv2beta1 "github.com/spacemeshos/api/release/go/spacemesh/v2beta1"
```

Comment:
You don't need to import this package at all - the test code is in the same package. You can just use `NewSmeshingServiceClient` instead of `spacemeshv2beta1.NewSmeshingServiceClient(conn)`